### ISSUE TITLE
The latest nightly test HF FP32 training accuracy test failed on eager_two_runs_differ

### ISSUE THREAD

**Issue by chuanqi129** [intel/torch-xpu-ops#151]
### 🐛 Describe the bug

Please refer https://github.com/intel/torch-xpu-ops/actions/runs/8722382297/job/23928124271, there are a lot of models accuracy failed with `eager_two_runs_differ`

============ Summary for huggingface float32 training accuracy ============
num_total: 46
num_passed: 5
num_failed: 41
pass_rate: **10.87%**

### Versions

torch-xpu-ops: 377696b906bdb30126b7c1edc90ee6078616d565
pytorch: https://github.com/etaf/pytorch-inductor-xpu/tree/e2e-baseline

**Comment by chuanqi129** [comment]
BTW, on-demand E2E test can be triggered by manually follow below steps, you can choose test which branch and configuration

![image](https://github.com/intel/torch-xpu-ops/assets/13608516/894700dd-6d1b-4210-8de2-828ae64c98d4)

**Comment by fengyuan14** [comment]
Fixing after rebasing RNG kernels.
